ovos-i2csound

```diff
@@ -108,6 +112,8 @@ main() {
         amixer -c "sndrpiproto" cset numid=13 on
         amixer -c "sndrpiproto" cset numid=9 on
         echo "Resetting Mark-1 faceplate"
+        # Check for permissions
+        chmod a+rw /dev/ttyAMA0
```

Comment:
ovos@mark1-dev:~ $ ll /dev/ttyAMA0
crw-rw---- 1 root dialout 204, 64 Feb 14 08:26 /dev/ttyAMA0

ovos@mark1-dev:~ $ sudo chmod a+rw /dev/ttyAMA0
ovos@mark1-dev:~ $ ll /dev/ttyAMA0
crw-rw-rw- 1 root dialout 204, 64 Feb 14 08:26 /dev/ttyAMA0

Reply:
This shouldn't be needed, but there was a permission issue when the script was run.  Even though it is run as root, and ovos is in `dialout` group.  Maybe because it runs so early?

Reply:
i dont think this is needed, works for me without it, i think the permission issue was a hardware problem